content/support-matrix/delfin/huawei/oceanstor/_index.md

```diff
@@ -0,0 +1,524 @@
+---
+title: OceanStor
+menuTitle: OceanStor
+description: "OceanStor Support Matrix"
+weight: 20
+disableToc: false
+tags: ["support matrix", "delfin", "drivers", "huawei"]
+---
+
+### Model and versions supported
+
+<table>
+   <thead>
+      <tr>
+         <th>Models</th>
+         <th>Management Software Versions</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Storage Management Software</td>
+         <td>OceanStor DeviceManager V100R002C20SPC100 or later</td>
+      </tr>
+   </tbody>
+</table>
+
+### Access information to register storage device
+
+<table>
+   <thead>
+      <tr>
+         <th>Access Type</th>
+         <th>Attributes</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td rowspan="4">REST</td>
+         <td>Host</td>
+         <td>OceanStor device manager host name supported with ipv4/ipv6</td>
+      </tr>
+      <tr>
+         <td>Port</td>
+         <td>OceanStor device manager port</td>
+      </tr>
+      <tr>
+         <td>Username</td>
+         <td>OceanStor device manager username</td>
+      </tr>
+      <tr>
+         <td>Password</td>
+         <td>OceanStor device manager password</td>
+      </tr>
+   </tbody>
+</table>
+
+### Alert management information
+
+<table>
+   <tbody>
+      <tr>
+          <th>SNMP trap processing</th> 
+          <td>Supported</td> 
+      </tr>
+      <tr>
+         <th>Query backend alert</th>
+         <td>Supported</td>
+      </tr>
+      <tr>
+         <th>Clear backend alert</th>
+         <td>Supported</td>
+      </tr>
+      <tr>
+         <th>SNMP versions for trap</th>
+         <td>V1/V2/V3</td>
+      </tr>
+      <tr>
+         <th>SNMP trap source</th>
+         <td>OceanStor Device</td>
+      </tr>
+      <tr>
+         <th>Query alert source</th>
+         <td>OceanStor Device Manager</td>
+      </tr>
+   </tbody>
+</table>
+
+### Performance registration information
+
+<table>
+   <thead>
+      <tr>
+         <th>Resource</th>
+         <th>Default Interval</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
```

Comment:
Table is empty. Remove if not available or applicable.

Reply:
@NajmudheenCT has to confirm